cmd/spire-agent/cli/api/fetch.go

```diff
@@ -78,127 +78,94 @@ func (f *FetchCLI) parseConfig(args []string) error {
 	fs := flag.NewFlagSet("fetch", flag.ContinueOnError)
 	c := &FetchConfig{}
 	fs.BoolVar(&c.silent, "silent", false, "Suppress stdout")
-	fs.StringVar(&c.spiffeID, "spiffeID", "", "Retrieve only a specific SPIFFE ID (optional)")
+	fs.IntVar(&c.timeout, "timeout", 1, "Number of seconds to wait for a response")
 	fs.StringVar(&c.socketPath, "socketPath", "/tmp/agent.sock", "Path to the Workload API socket")
 	fs.StringVar(&c.writePath, "write", "", "Write SVID data to the specified path (optional)")
 
 	f.config = c
 	return fs.Parse(args)
 }
 
-func (f *FetchCLI) fetchBundles(ctx context.Context, c workload.WorkloadClient) (*workload.Bundles, error) {
-	var resp *workload.Bundles
-	var err error
+func (f *FetchCLI) fetchX509SVID(c workload.SpiffeWorkloadAPIClient) (*workload.X509SVIDResponse, error) {
```

Comment:
nit: Some times the functions are defined using (f *FetchCLI) and another times using (f FetchCLI), maybe we should use only one of the options.

Reply:
The ones that don't have a pointer receiver don't need any state... I thought maybe they could be useful somewhere else, but also perhaps they don't make much sense to have a receiver at all in that case... maybe I should just define them separately?

Reply:
No, I think its fine. Was just something I saw and wanted to let you know just in case.